list-pr-issue.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
Please add a license header

Reply:
> I do not see any obvious problems with the code. Are there any coding styles or other things to be checked or is a quick glance ok for you?

quick glance is ok - this is just maintenance/utility code